javascript/javascript1/week4/voice.js

```diff
@@ -0,0 +1,54 @@
+const user = " Michael";
+const todos = [];
+
+function getReply(command) {
+  if (command === "Hello my name is" + user) {
+    return "nice to meet you" + user;
+  }
+  if (command === "What is my name?") {
+    return "your name is" + user;
+  }
+
+  if (command === "Add fishing to my todo") {
+    todos.push("fishing");
+    return "fishing is added to your todo list";
+  }
+  if (command === "Add singing in the shower to my todo") {
+    todos.push("singing in the shower");
+    return "added singing in the shower to a list of todos";
+  }
+  if (command === "Remove fishing from my todo") {
+    return "Removed fishing from your todo";
+  }
+  if (command === "What is on my todo?") {
+    return todos;
+  }
```

Comment:
What if user wants to enter other activity than fishing to todo? does this still add activity to todo?

Suggestions:
use split( ), slice( ), startsWith( ), endsWith( ) functions where-ever required to make it work for varied inputs.
Make it dynamic, user should be able to add  or remove any activity.